pkg/kv/dist_sender.go

```diff
@@ -59,6 +60,36 @@ const (
 	defaultSenderConcurrency = 500
 )
 
+var (
+	// DistSender Metrics
+	metaTotalSends         = metric.Metadata{Name: "distsender.totalSends"}
```

Comment:
we do, but I was going to refrain from proposing different names until I understood what each metric is and how they relate to each other.
we've definitely stayed away from camelcase though.